model-optimizer/mo/front/tf/loader.py

```diff
@@ -140,7 +140,7 @@ def get_output_node_names_list(graph_def, user_defined_output_node_names_list: l
 
```

Comment:
What about file `model-optimizer/extensions/back/SpecialNodesFinalization.py`, line 82?

Reply:
what are there? I dont see warnings there.

Reply:
My bad. There is already "r" in the beginnig.